modules/Data.py

```diff
@@ -9,6 +10,7 @@ def init(api1, log1):
     global api, log
     api = api1
     log = log1
+    get_lending_currencies()
```

Comment:
Data.init is called outside the exception handling loop and if this call fails - the exception won't be handled.